<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoshow</class>
 <widget class="QWidget" name="videoshow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="m_pLb_VideoPlayer">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>0</y>
     <width>931</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLb_FileIndTotal">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: white;           
    background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLb_FileName">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: white;           
    background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Return">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>700</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/return.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Del_File">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>690</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/delfile.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Left">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>80</width>
     <height>146</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Right">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>300</y>
     <width>80</width>
     <height>146</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pLb_VideoPlayer_Play">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/videoflag.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_nosel.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>m_pLb_VideoPlayer</zorder>
  <zorder>m_pLb_FileIndTotal</zorder>
  <zorder>m_pLb_FileName</zorder>
  <zorder>m_pBtn_Return</zorder>
  <zorder>m_pBtn_Del_File</zorder>
  <zorder>m_pBtn_Left</zorder>
  <zorder>m_pBtn_Right</zorder>
  <zorder>m_pLb_VideoPlayer_Play</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
